styles.css

```diff
@@ -81,8 +88,52 @@ header {
 	text-transform: uppercase;
 }
 
+main {
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: space-evenly;
+}
+
 header img {
 	width: 100%;
-	margin-bottom: -100px;
-	
+}
+
+.portfolio {
+	width: 55%;
+	height: 1000px;
+	padding: 20px;
+	overflow: scroll;
+}
+
+.portfolio-card {
+	border: 1px solid;
+	padding: 20px;
+	margin: 20px 0px;
+}
+
+.portfolio-card img{
+	width: 40%;
+}
+
+.about {
+	width: 30%;
+	padding: 20px;
+	height: 1000px;
+}
+
+.about img {
+	width: 60%;
+}
+
+.contact {
+	background-color: #6a6c6e;
+	margin: 0;
+	padding: 20px 60px;
+}
+
+.social-link-icons {
+	width: 200px;
+	display: flex;
+	flex-flow: row warp;
```

Comment:
typo on 'wrap', but do you want this? The page cant get small enough to make them wrap, don't think it's required